inline_scan.sh

```diff
@@ -140,6 +143,11 @@ get_and_validate_analyzer_options() {
 
     SYSDIG_SCANNING_URL="${SYSDIG_BASE_SCANNING_URL}"/api/scanning/v1
     SYSDIG_ANCHORE_URL="${SYSDIG_SCANNING_URL}"/anchore
+
+    if [[ "${l_flag:-}" ]]; then
+        INSECURE_CURL='--basic'
```

Comment:
The name `INSECURE_CURL` doesn't really describe what it contains. How about something like `CURL_AUTH`?